src/repository/screens/repository.screen.js

```diff
@@ -158,168 +162,212 @@ class Repository extends Component {
         {loader}
-
-        <ParallaxScroll
-          renderContent={() => {
-            if (isPendingRepository && !initalRepository) {
-              return <LoadingRepositoryProfile />;
-            }
-
-            return (
-              <RepositoryProfile
-                repository={isPendingRepository ? initalRepository : repository}
-                starred={
-                  isPendingRepository || isPendingCheckStarred ? false : starred
-                }
-                navigation={navigation}
-              />
-            );
-          }}
-          stickyTitle={repository.name}
-          showMenu={!isPendingRepository && !isPendingCheckStarred}
-          menuAction={() => this.showMenuActionSheet()}
-          navigation={navigation}
-          navigateBack
+        <ScrollView
+          refreshControl={
+            <RefreshControl
+              refreshing={isPendingRepository}
+              onRefresh={this.fetchRepoInfo}
+            />
+          }
         >
-          {initalRepository &&
-            !initalRepository.owner &&
-            isPendingRepository &&
-            <SectionList title="OWNER">
-              <LoadingUserListItem />
-            </SectionList>}
+          <ParallaxScroll
+            renderContent={() => {
+              if (isPendingRepository && !initalRepository) {
+                return <LoadingRepositoryProfile />;
+              }
 
-          {!(initalRepository && initalRepository.owner) &&
-            (repository && repository.owner) &&
-            !isPendingRepository &&
-            <SectionList title="OWNER">
-              <UserListItem user={repository.owner} navigation={navigation} />
-            </SectionList>}
+              return (
+                <RepositoryProfile
+                  repository={
+                    isPendingRepository ? initalRepository : repository
+                  }
+                  starred={
+                    isPendingRepository || isPendingCheckStarred
+                      ? false
+                      : starred
+                  }
+                  navigation={navigation}
+                />
+              );
+            }}
+            stickyTitle={repository.name}
+            showMenu={!isPendingRepository && !isPendingCheckStarred}
+            menuAction={() => this.showMenuActionSheet()}
+            navigation={navigation}
+            navigateBack
+          >
+            {initalRepository &&
+              !initalRepository.owner &&
+              isPendingRepository &&
+              <SectionList title="OWNER">
+                <LoadingUserListItem />
+              </SectionList>}
 
-          {initalRepository &&
-            initalRepository.owner &&
-            <SectionList title="OWNER">
-              <UserListItem
-                user={initalRepository.owner}
-                navigation={navigation}
-              />
-            </SectionList>}
+            {!(initalRepository && initalRepository.owner) &&
```

Comment:
We need to abstract this to a method somehow. This will make maintainability quite difficult.